projects/npm-tools/packages/npm-scripts/src/scripts/webpack.js

```diff
@@ -51,6 +51,23 @@ function escapeLiteralString(str) {
 function withWebpackConfig(filename, callback) {
 	const webpackConfigPath = path.resolve(filename);
 
+	let federation;
+
+	if (FEDERATION_CONFIG) {
+		if (typeof FEDERATION_CONFIG === 'boolean') {
+			federation = '{}';
+		}
+		else if (typeof FEDERATION_CONFIG === 'object') {
+			federation = '{\n';
```

Comment:
Is there a clean way to generate a JS (not JSON) valid representation of an object?

Reply:
Why can't you use JSON (which is JS)?

Reply:
:thinking: That's true. However I had issues when using JSON in the past but now that I realize it should work I don't know why :shrug: .